packages/components-js/src/examples/flyout-example.html

```diff
@@ -0,0 +1,21 @@
+<p-button type="button" aria="{ 'aria-haspopup': 'dialog' }">Open Flyout</p-button>
+
+<p-flyout open="false">
+  <div slot="header">
+    <p-heading tag="h5" size="large">Sticky Heading</p-heading>
+    <p-text size="small">Sticky header text</p-text>
+  </div>
+  <p-text>Some Content</p-text>
+  <div slot="footer">
+    <p-button>Footer Button</p-button>
+  </div>
+  <p-text slot="secondary-content">Some Secondary Content</p-text>
+</p-flyout>
+
+<script>
+  const flyout = document.querySelector('p-flyout');
+  flyout.addEventListener('dismiss', () => {
+    flyout.open = false;
+  });
+  document.querySelector('p-button').addEventListener('click', () => (flyout.open = true));
```

Comment:
Let's add a function body here for better readability and consistency with previous event listener since it is for user facing. documentation.